src/apprt/gtk/Window.zig

```diff
@@ -575,6 +579,72 @@ pub fn sendToast(self: *Window, title: [:0]const u8) void {
     c.adw_toast_overlay_add_toast(@ptrCast(toast_overlay), toast);
 }
 
+fn displayCrashReportsNoticeIfNeeded(self: *Window, app: *App, parent_widget: *c.GtkWidget) !void {
+    if (App.crash_reports_notice_dismissed) {
+        return;
+    }
+
+    const crash_dir = try crash.defaultDir(self.app.core_app.alloc);
+    defer self.app.core_app.alloc.free(crash_dir.path);
+    var it = try crash_dir.iterator();
+    defer it.deinit();
+
+    if (try it.next()) |_| {
+        const warning_text = try std.fmt.allocPrintZ(self.app.core_app.alloc,
+            \\ ⚠️ There are existing crash reports located at <a href="file://{s}">{s}</a>.
+            \\ If possible, <a href="https://github.com/ghostty-org/ghostty?tab=readme-ov-file#crash-reports">report them to the developers</a> to help improve the application.
+        , .{ crash_dir.path, crash_dir.path });
```

Comment:
FYI you can avoid repeating the parameter by specifying `{0s}` in the format string